From 634bf2d0de01d470f09388ae927632a1637370ae Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:04:40 -0400
Subject: [powerpc] powernv: Enable detection of legacy UARTs

Message-id: <1410545655-205645-52-git-send-email-dzickus@redhat.com>
Patchwork-id: 94112
O-Subject: [RHEL7 PATCH 051/626] powerpc/powernv: Enable detection of legacy UARTs
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 8a05dd851410b3a299421a6c6975c081d3c805a8
Author: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Date:   Mon Jul 15 13:03:15 2013 +1000

    powerpc/powernv: Enable detection of legacy UARTs

    Legacy UARTs can exist on PowerNV, memory-mapped ones on PCI
    or IO based ones on the LPC bus.

    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/platforms/powernv/Kconfig b/arch/powerpc/platforms/powernv/Kconfig
index 6529587..6fae5eb 100644
--- a/arch/powerpc/platforms/powernv/Kconfig
+++ b/arch/powerpc/platforms/powernv/Kconfig
@@ -8,6 +8,7 @@ config PPC_POWERNV
  select PPC_PCI_CHOICE if EMBEDDED
  select EPAPR_BOOT
  select PPC_INDIRECT_PIO
+ select PPC_UDBG_16550
  default y
 
 config POWERNV_MSI
-- 
1.7.1